biobricks/dvc_fetcher.py

```diff
@@ -158,7 +108,7 @@ def _link_cache_to_brick(self, cache_path, brick_path):
         else:
             os.symlink(cache_path, brick_path)
         
-    def fetch_outs(self, brick, prefixes=['brick/', 'data/']) -> tuple[list[dict], int]:
+    def fetch_outs(self, brick, prefixes=['brick/', 'data/', 'list/']) -> tuple[list[dict], int]:
```

Comment:
We shouldn't need `list/` here. The bricks should be eventually all be standardized to use `brick/` as the output.